geometry_msgs/Point centroid
float32 confidence
